docs/getting-started/overview.rst

```diff
@@ -1,6 +1,113 @@
+
+With all of the above, .NET Core represents a very generic, cross-platform and modern 
+runtime that is capable of supporting multiple workloads, from cloud services, to desktop 
+applications to CLI tools. 
+
+.NET Framework
+--------------
+
+The .NET Framework is the premier implemetentation of the .NET Platform available for 
+Windows server and client developers. It is a very powerful, very mature framework, with 
+a huge class library (known as the **Framework Class Librariries**) that supports 
+a wide variety of applications and solutions on Windows. 
+
+There are additional stacks built on top the .NET Framework that allow developers 
+to build applications ranging from console applications, across rich client (WPF) 
+applications to scalable web applications. 
+
+`Windows Forms <https://msdn.microsoft.com/en-us/library/dd30h2yb(v=vs.110).aspx>`_ 
+and `Windows Presentation Foundation (WPF) <https://msdn.microsoft.com/en-us/library/ms754130(v=vs.110).aspx>`_ 
+are two User Interface (UI) stacks that allow you to build desktop applications for Windows 
+operating system. Windows Forms' strength is in its rich support for common databinding as well as 
+access to Windows' native user interface controls. WPF, on the other hand, allows you to exercise 
+much more control over the look and feel of your application. Both of them allow for building very 
+rich desktop applications that run on Windows, and you should pick the one that is suited for your 
+use case. 
+
+`Windows Communication Foundation (WCF) <https://msdn.microsoft.com/en-us/library/ms731082(v=vs.110).aspx>`_ 
+is a set of libraries that comprise the middleware services stack on .NET Framework. 
+It allows you to create services that can communicate through various supported 
+protocols using various data formats, and that can be hoster in any process 
+you choose. This leads to one of the major features of WCF: your services are 
+not tied to any particular hosting strategy or approach.
+
+**ASP.NET** is the .NET web framework. Being a very rich framework, it has several distinct pieces 
+which are used to produce modern and high-performance web applications. 
+`ASP.NET Web Forms <http://www.asp.net/web-forms>`_ is a set of tools geared primarily towards 
+developer productivity, allowing quick turnaround on web applications with a drag-and-drop surface 
+reusing web controls for everything from loging in to data binding. 
+`ASP.NET MVC <http://www.asp.net/mvc>`_ allows for a different approach, one that gives you greater 
+control over the entire pipeline, from HTTP layer to the user interface. 
+`ASP.NET WebAPI <http://www.asp.net/web-api>`_ is a convention-based framework for creating REST 
+services. It allows you to stand up a REST endpoint extremely fast. Finally, 
+`SignalR <http://www.asp.net/signalr>`_ allows you to provide push-based communication to your web 
+applications using `WebSocket <https://en.wikipedia.org/wiki/WebSocket>`_ protocol. 
+
+
+.NET Native
```

Comment:
OK, so let's go with this document being similar in nature to http://dotnet.github.io/core/about/overview.html, that is, have this be not an overview of .NET Editions but of the entire landscape. This would also help us have one "version of truth", as then the linked page would be replaced with whatever is in this, RTD, document, Would that make sense?